doc/getting_started/getting_started.rst

```diff
@@ -5,7 +5,7 @@ This tutorial will install MoveIt 2 and create a workspace sandbox to run the tu
 
 Install ROS 2 and Colcon
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-`Install ROS 2 Foxy <https://docs.ros.org/en/foxy/Installation.html>`_.
```

Comment:
I could not for the life of me figure out how to get the DISTRO macro to resolve.

Reply:
Me neither.  I know the ros2 docs use them and I copied them from there but I haven't figured out what we are doing wrong with them.  This is a reasonable solution for now as it is how we resolve many of our other links.

Reply:
The knowledge that ros2 docs do it was enough to point me in the right direction.